netty/src/main/java/io/grpc/netty/NettyClientTransport.java

```diff
@@ -113,6 +116,24 @@ class NettyClientTransport implements ConnectionClientTransport {
       Runnable tooManyPingsRunnable, TransportTracer transportTracer, Attributes eagAttributes,
       LocalSocketPicker localSocketPicker, ChannelLogger channelLogger,
       boolean useGetForSafeMethods) {
+    this(address,channelFactory, channelOptions, group, negotiator, autoFlowControl,
+        flowControlWindow, maxMessageSize, maxHeaderListSize, keepAliveTimeNanos,
+        keepAliveTimeoutNanos, keepAliveWithoutCalls, authority, userAgent, tooManyPingsRunnable,
+        transportTracer, eagAttributes, localSocketPicker, channelLogger, useGetForSafeMethods,
+        Ticker.systemTicker());
+  }
+
+  NettyClientTransport(
```

Comment:
Do we really need another constructor? Is it too onerous to modify the existing one?

Reply:
I was concerned about compatibility for our users.  I've updated all references and removed the old constructor.